variables.tf

```diff
@@ -1,5 +1,45 @@
-
 variable "aws_account_id" {
   type        = string
   description = "The id of the aws account we want to extract the metadata from."
 }
+
+variable "stage" {
+  type        = string
+  description = "Stage of the outputs, e.g. dev, tst, acc, prd"
+}
+
+variable "deployment" {
```

Comment:
Will possibly be removed and deployment set per baseline - requires knowledge of which deployment is needed by baseline, but is consistent - no strange errors, you get what you ask for